### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by MannyPeterson**
I am moving the HeliOS library from my personal GitHub account (https://github.com/mannypeterson) to a GitHub organization (https://github.com/heliosproj) which will remain the home for the project (library) for the foreseeable future. Thank you, Manny Peterson

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @MannyPeterson. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Comment by MannyPeterson** [comment]
> Hello @MannyPeterson. I am confirming your request.
> 
> Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.
> 
> After that, you can monitor the indexing of the library via its dedicated logs web page. Information about that here: https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

Thank you @per1234

**Review by MatteoPologruto — APPROVED** [review]

**Comment by MatteoPologruto** [comment]
Done:
```
2023/03/22 16:53:56 Loaded 5648 libraries from DB
Changing URL of library HeliOS from https://github.com/MannyPeterson/HeliOS.git to https://github.com/heliosproj/HeliOS.git
Success!
```

**Pull request merged**
